xds/src/main/java/io/grpc/xds/ClientXdsClient.java

```diff
@@ -547,13 +541,38 @@ static void validateCommonTlsContext(
         }
       }
     }
-    if (rootCaInstanceName != null) {
-      if (certProviderInstances == null || !certProviderInstances.contains(rootCaInstanceName)) {
-        throw new ResourceInvalidException(
-            "ValidationContextProvider instance name '" + rootCaInstanceName
-                + "' not defined in the bootstrap file.");
+  }
+
+  private static String getIdentityCertInstanceName(CommonTlsContext commonTlsContext) {
+    if (commonTlsContext.hasTlsCertificateProviderInstance()) {
+      return commonTlsContext.getTlsCertificateProviderInstance().getInstanceName();
+    } else if (commonTlsContext.hasTlsCertificateCertificateProviderInstance()) {
```

Comment:
This is deprecated, are we still supporting it?

Reply:
Yes, until Traffic Director is updated to make this change we will have to support the older/deprecated fields